domain-models-runtime/src/test/java/org/folio/rest/persist/PgUtilIT.java

```diff
@@ -555,6 +589,200 @@ public void responseWithExceptionInFailResponseMethod(TestContext testContext) t
+         }
+         for (int i=0; i<4; i++) {
+           JsonObject instance = json.getJsonObject(4 + i);
+           assertThat(instance.getString("name"), is("d foo " + (i + 1)));
+         }
+      } );
+
+
+    // // "b foo", offset=1, limit=20
+    // json = searchForInstances("title=b sortBy title/sort.ascending", 1, 20);
+    searchForData("name=b sortBy name/sort.ascending", 1, 20,"user","testtenant",testContext,columnName,optimizdSQLSize,reply -> {
+        //handle return and
+        if (reply.failed()) {
+      	  testContext.fail(reply.cause());
+        }
+        JsonArray json = reply.result();
+        assertThat(json.size(), is(4));
+       
+         for (int i=0; i<4; i++) {
+           JsonObject instance = json.getJsonObject(i);
+           assertThat(instance.getString("name"), is("b foo " + (1 + i + 1)));
+         }
+      } );
+
+
+    // // sort.descending, offset=1, limit=3
+    // json = searchForInstances("title=foo sortBy title/sort.descending", 1, 3);
+    searchForData("name=foo sortBy name/sort.ascending", 1, 3,"user","testtenant",testContext,columnName,optimizdSQLSize,reply -> {
+        //handle return and
+        if (reply.failed()) {
+      	  testContext.fail(reply.cause());
+        }
+        JsonArray json = reply.result();
+        assertThat(json.size(), is(3));
+       
+        
+         for (int i=0; i<3; i++) {
+           JsonObject instance = json.getJsonObject(i);
+           assertThat(instance.getString("name"), is("d foo " + (4 - i)));
+         }
+      } );
+
+
+    // // sort.descending, offset=6, limit=3
+    // json = searchForInstances("title=foo sortBy title/sort.descending", 6, 3);
+      searchForData("name=foo sortBy name/sort.ascending" ,6,3,"user","testtenant",testContext,columnName,optimizdSQLSize,  reply -> {
+        //handle return and
+        if (reply.failed()) {
+      	  testContext.fail(reply.cause());
+        }
+        JsonArray json = reply.result();
+        assertThat(json.size(), is(3));
+       
+        
+         for (int i=0; i<3; i++) {
+           JsonObject instance = json.getJsonObject(i);
+           assertThat(instance.getString("name"), is("b foo " + (4 - i)));
+         }
+    });
+
+}
+  private void searchForData(String cql,int limit, int offset, String table,String tennant,TestContext testContext, String columnName,int optimizdSQLSize,
+      Handler<AsyncResult<JsonArray>>  handler ) {
+    String optimizedSQL = "";
+    PreparedCQL pCQL = null;
+    CQLWrapper wrapper = null;
+    try {
+      wrapper = PgUtil.createCQLWrapper(cql, limit, offset, Arrays.asList(schema + "." + table + ".jsonb "));
+      pCQL = new PreparedCQL(table, wrapper );
+      optimizedSQL = PgUtil.optimizedSql(pCQL, tennant, 6, 3, columnName, optimizdSQLSize);
+    } catch(FieldException fe) {
+      testContext.fail(fe.getMessage());
+    } catch(Exception e) {
+      testContext.fail(e.getMessage());
+    }
+    log.error("optimized sql is " + optimizedSQL);
```

Comment:
Why is this an error? If it isn't, can you log it with a suitable non-error level?